cli/command/network/connect.go

```diff
@@ -56,7 +63,23 @@ func runConnect(dockerCli *command.DockerCli, options connectOptions) error {
 		},
 		Links:   options.links.GetAll(),
 		Aliases: options.aliases,
+		DriverOpts: driverOpts,
 	}
 
 	return client.NetworkConnect(context.Background(), options.network, options.container, epConfig)
 }
+
+func convertDriverOpt(opts []string) (map[string]string, error) {
+	driverOpt := make(map[string]string)
+	for _, opt := range opts {
+		parts := strings.SplitN(opt, "=", 2)
+		if len(parts) != 2 {
+			err := fmt.Errorf("invalid key value pair format in driver options")
+			return nil, err
+		}
+		key := strings.TrimSpace(strings.ToLower(parts[0]))
+		value := strings.TrimSpace(strings.ToLower(parts[1]))
```

Comment:
yes I thought of that as well. But I saw that the other options like port were converting to lower case. So I decided to maintain the uniformity.